package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ultravox-client",
-  "version": "0.2.15",
+  "version": "0.3.1",
```

Comment:
it adds functionality so that's why i bumped minor version

Reply:
Right, but until we have a non-zero major version, bumping the minor version will be treated as a breaking change by dependency managers.  (Or at least it should be. It would be for Python or Flutter. I'm never sure with JS.)

Reply:
js is the wild west :). i switched to `0.2.16`